src/vcQuadTree.h

```diff
@@ -48,7 +52,7 @@ struct vcQuadTreeNode
   bool rendered;
 
   // cached
-  udDouble2 worldBounds[4]; // corners
+  udDouble2 worldBounds[9]; // corners
```

Comment:
Does this comment still apply? Just want to confirm that they're still _corners_.